.github/workflows/labeler.yml

```diff
@@ -1,6 +1,7 @@
 name: 'Pull Request Labeler'
 on:
-  - pull_request_target
+  pull_request_target:
+    types: [opened]
```

Comment:
Hmm I think this one is okay.
<img width="675" alt="image" src="https://github.com/user-attachments/assets/d30254f3-0069-4a4e-be5b-31453112129b" />